### PR TITLE
samples cleanup

### PR DESCRIPTION
**Pull request by kirill-pyatunin**
- redundant usings removed
- modifiers added
- a few methods were renamed based on naming convention
- general cleanup

### CODE REVIEW

File: Samples/NotifyUsers/AppBackend/Controllers/NotificationsController.cs [hunk 1]
```diff
@@ -57,13 +54,15 @@ public async Task<HttpResponseMessage> Post(string pns, [FromBody]string message
...
             {
-                if (!((outcome.State == Microsoft.Azure.NotificationHubs.NotificationOutcomeState.Abandoned) ||
-                    (outcome.State == Microsoft.Azure.NotificationHubs.NotificationOutcomeState.Unknown)))
-                {
-                    ret = HttpStatusCode.OK;
-                }
+                return Request.CreateResponse(ret);
+            }
+
+            if (!(outcome.State == Microsoft.Azure.NotificationHubs.NotificationOutcomeState.Abandoned ||
```
Comment (igprya) [inline Samples/NotifyUsers/AppBackend/Controllers/NotificationsController.cs]: It's more of a personal preference but I would have moved the `||` operator to the beginning of the next line, something like this:
```
if (!((outcome.State == Microsoft.Azure.NotificationHubs.NotificationOutcomeState.Abandoned) 
    || (outcome.State == Microsoft.Azure.NotificationHubs.NotificationOutcomeState.Unknown)))
```
And it seems that this statement might be reversed to get rid of the `!` at the beginning and consequential parenthesis pair, but it's only a guess that might be incorrect. 🙂

File: Samples/NotifyUsers/AppBackend/AuthenticationTestHandler.cs [hunk 2]
```diff
@@ -45,13 +41,13 @@ public class AuthenticationTestHandler : DelegatingHandler
...
 
-        private bool verifyUserAndPwd(string user, string password)
+        private static bool VerifyUserAndPwd(string user, string password)
         {
             // This is not a real authentication scheme.
             return user == password;
         }
 
-        private Task<HttpResponseMessage> Unauthorized()
+        private static Task<HttpResponseMessage> Unauthorized()
```
Comment (brannon) [inline Samples/NotifyUsers/AppBackend/AuthenticationTestHandler.cs]: Can you update this to return `Task.FromResult(response)` ? And any other place where we use `TaskCompletionSource` in this manner?

File: Samples/SendRestExample/SendRestExample/ConnectionStringUtility.cs [hunk 3]
```diff
@@ -15,26 +15,34 @@ public ConnectionStringUtility(string connectionString)
             //Parse Connectionstring
             char[] separator = { ';' };
             string[] parts = connectionString.Split(separator);
-            for (int i = 0; i < parts.Length; i++)
+            foreach (var t in parts)
```
Comment (brannon) [inline Samples/SendRestExample/SendRestExample/ConnectionStringUtility.cs]: Convention is to call the element variable `part` if the collection is the plural `parts`.

File: Samples/SendRestExample/SendRestExample/ConnectionStringUtility.cs [hunk 4]
```diff
@@ -15,26 +15,34 @@ public ConnectionStringUtility(string connectionString)
...
         {
             string targetUri = Uri.EscapeDataString(uri.ToLower()).ToLower();
 
             // Add an expiration in seconds to it.
             long expiresOnDate = DateTime.Now.Ticks / TimeSpan.TicksPerMillisecond;
             expiresOnDate += minUntilExpire * 60 * 1000;
-            long expires_seconds = expiresOnDate / 1000;
-            String toSign = targetUri + "\n" + expires_seconds;
+            long expiresSeconds = expiresOnDate / 1000;
+            String toSign = targetUri + "\n" + expiresSeconds;
```
Comment (brannon) [inline Samples/SendRestExample/SendRestExample/ConnectionStringUtility.cs]: Let's update this method to use `var` for the variables. The types are obvious from the usage.

File: Samples/SendRestExample/SendRestExample/Program.cs [hunk 5]
```diff
@@ -72,7 +72,7 @@ private static async Task<string> SendNativeNotificationREST(string hubname, str
             string uri = connectionSaSUtil.Endpoint + hubname + "/" + hubResource + apiVersion;
 
             // 10 min expiration
-            string SasToken = connectionSaSUtil.getSaSToken(uri, 10);
+            string SasToken = connectionSaSUtil.GetSaSToken(uri, 10);
```
Comment (brannon) [inline Samples/SendRestExample/SendRestExample/Program.cs]: `SasToken` should use camel case: `sasToken`

File: Samples/SendRestExample/SendRestExample/Program.cs [hunk 6]
```diff
@@ -113,7 +113,7 @@ private static async Task<string> SendNativeNotificationREST(string hubname, str
             char[] seps1 = { '?' };
             char[] seps2 = { '/' };
 
-            if ((int)response.StatusCode != 201)
+            if (response != null && (int)response.StatusCode != 201)
```
Comment (brannon) [inline Samples/SendRestExample/SendRestExample/Program.cs]: What is the type of `response.StatusCode`? If it's an enum, can we just use the enum directly?

**Pull request merged**